{
    "exceptions": {
        "command_error": {
            "message": "Erro ao executar {function_name} na entidade {entity_id}"
        },
        "entry_cannot_connect": {
            "message": "Erro ao ligar a {host}:{port}"
        }
    }
}